### PR TITLE
Revert "Add june 11th webinar banner"

### PR DESCRIPTION
**Pull request by curquiza**
Reverts meilisearch/meilisearch#4691

### CODE REVIEW

**Comment by CaroFG** [comment]
Just an idea, we could say: "**We hosted** an event..." and link to the blog post: https://blog.meilisearch.com/heres-what-you-missed-q2-2024/

**Comment by curquiza** [comment]
Feel free to open a new PR @Strift or @CaroFG 👌 or to make suggestion on this one

**Review by Strift — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -25,12 +25,6 @@
 
 <p align="center">⚡ A lightning-fast search engine that fits effortlessly into your apps, websites, and workflow 🔍</p>
```
Comment (CaroFG) [inline README.md:27]:
```suggestion

---

### 🔥 On June 11th, we hosted a live product demo and Q&A about AI search, query suggestions, and v1.9. If you missed the event, don't worry—you can catch up with all the highlights and details in our [recap](https://blog.meilisearch.com/heres-what-you-missed-q2-2024/?utm_campaign=oss&utm_source=github&utm_medium=meilisearch)!

---
```

**Comment by Strift** [comment]
Hey, 

I think it's fine to advertise upcoming events, but I fear promoting past events will take us down a path where the README feels like an advertising billboard. Let's keep this PR focused on reverting the banner. 

I'm open to the discussion, but I agree with @curquiza's comment—let's take this out of this PR.

**Pull request merged**